### PR TITLE
Suggested edits to metrics deep-dive

### PR DESCRIPTION
**Pull request by meiqimichelle**
Some suggested edits to the metrics deep-dive after the A+ start from @lanzafame 🎉

### CODE REVIEW

File: DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 # IPFS metrics and usage statistics
-Open problems around metrics and usage statistics in IPFS can be grouped as follows:
+
+## Problem space
+1. Certain user groups (such as scientific research organizations, and binary/library package managers) will not adopt IFPS unless they can track metrics and usage statistics in as good or better ways as they do today. Today, download count is used as a proxy for usage. How could IPFS provide even better metrics/stats?
```
Comment (momack2) [inline DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md]:
```suggestion
1. Certain user groups (such as scientific research organizations, and binary/library package managers) will not adopt IPFS unless they can track metrics and usage statistics in as good or better ways as they do today. Today, download count is used as a proxy for usage. How could IPFS provide even better metrics/stats?
```

whoops ☺️

File: DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
 # IPFS metrics and usage statistics
-Open problems around metrics and usage statistics in IPFS can be grouped as follows:
+
+## Problem space
+1. Certain user groups (such as scientific research organizations, and binary/library package managers) will not adopt IPFS unless they can track metrics and usage statistics in as good or better ways as they do today. Today, download count is used as a proxy for usage. How could IPFS provide even better metrics/stats?
+1. How are applications using IPFS features today? How are those applications themselves being used?
+
+## Goal
+By the end of the session, we'd like to have a prioritized list of features that would most help applications built on top of IPFS, and provide that list to the IPFS development team to help focus their time and energy on highest-value work.
```
Comment (alanshaw) [inline DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md]: ...and an idea or proposal for how they might be implemented?

**Comment by alanshaw** [comment]
@lanzafame ping - is this ok with you?

File: DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md (hunk 2 above)
Comment (lanzafame) [reply DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md]: hmm I agree with this. I don't want to come out of this deep dive with a list of nice to haves that aren't even remotely possible in a technical sense.

**Review by lanzafame — CHANGES_REQUESTED** [review]
@meiqimichelle Could you please adjust the goal to include @alanshaw comment?

File: DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md (hunk 2 above)
Comment (alanshaw) [inline DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md]:
```suggestion
By the end of the session, we'd like to have a prioritized list of features that would most help applications built on top of IPFS and an accompanying set of ideas/proposals for how they might be implemented.

We should be able to provide that list to the IPFS development team to help focus their time and energy on highest-value work.
```

@lanzafame @meiqimichelle how's that?

Comment (meiqimichelle) [reply DEEP_DIVES/44-ipfs-metrics-and-usage-statistics.md]: I accepted @alanshaw 's suggestion -- thank you!

**Review by alanshaw — APPROVED** [review]

**Pull request merged**